internal/chart/application_chart.go

```diff
@@ -70,8 +73,8 @@ type app struct {
 	// For example, "spec.rules" of an Ingress object must contain at least one rule.
 	IsAccessible bool   `json:"isAccessible"`
 	Group        string `json:"group"`
-
-	Service *gatewayService
+	Secret       string `json:"secret"`
```

Comment:
not sure but are "secretName" and json:"secret" same things? 
If yes how about have similar naming for both of them? :)

Reply:
Good call. Changed to `secretName`.